pkg/nfd-master/nfd-master.go

```diff
@@ -519,6 +518,22 @@ func isNamespaceDenied(labelNs string, wildcardDeniedNs map[string]struct{}, nor
 	return false
 }
 
+// if ns != nfdv1alpha1.ExtendedResourceNs && ns != nfdv1alpha1.FeatureLabelNs &&
+// !strings.HasSuffix(ns, nfdv1alpha1.ExtendedResourceSubNsSuffix) && !strings.HasSuffix(ns, nfdv1alpha1.FeatureLabelSubNsSuffix)
```

Comment:
this comment is rather strange

Reply:
ups, yeah I had it up there as a guide when writing the func